tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -816,20 +819,26 @@ used, and how the editor should allow users to modify it.
     uniform vec4 other_color : source_color = vec4(1.0); // Default values go after the hint.
     uniform sampler2D image : source_color;
 
-It's important to understand that textures *that are supplied as color* require
-hints for proper sRGB -> linear conversion (i.e. ``source_color``), as Godot's
-3D engine renders in linear color space. If this is not done, the texture will
-appear washed out.
+.. admonition:: Source Color
 
-.. note::
+    Any texture which contains *sRGB color data* requires a ``source_color`` hint
+    in order to be correctly sampled. This is because Godot renders in linear
+    color space, but some textures contain sRGB color data. If this hint is not
+    used, the texture will appear washed out.
+
+    Albedo and color textures should have a ``source_color`` hint. Normal,
```

Comment:
```suggestion
    Albedo and color textures should typically have a ``source_color`` hint. Normal,
```
I suggest softening this as it isn't always true, it depends on the source image itself. It's surprising, but once in awhile someone uses a texture with linear space colors in it